### DISCUSSION TITLE
Is there any way to provide a start adornment to the input of combobox?

### DISCUSSION THREAD

**Discussion by SaavanNanavati** [ariakit/ariakit#1953] [Questions]
So let's say I have a multiselect combobox. Basically, I want to display the selected values as chips inside of the input similar to how it looks here: https://mui.com/material-ui/react-autocomplete/#multiple-values. In this case though, I'll have to manually render the chips myself as a start adornment to the input. I am able to render the chips but not as a start adornment. Do you have a way to do this, to basically make it look like the MUI link I sent? Thank you.

**Comment by diegohaz** [comment]
What do you mean by "start adornment"? What's missing in the CodeSandbox I shared on https://github.com/ariakit/ariakit/discussions/1929#discussioncomment-3817254?
